esp32-srne-asf-example.yaml

```diff
@@ -772,13 +772,23 @@ switch:
   - platform: modbus_controller
     use_write_multiple: true
     modbus_controller_id: inverter0
-    name: "Alarms"
+    name: "Buzzer"
```

Comment:
```suggestion
    name: "Acoustic Alarm"
```

Reply:
I wonder if _Alarm_ in singular will be somehow confusing. What about _Acoustic Alarms_? In fact, I think I preferred _Audible Alarms_ rather than _Acoustic_.

Reply:
Good point!

Reply:
`Audible Alarms` is fine for me too.